source/administrator-guide/configuring-the-catalog/user-interface-configuration.rst

```diff
@@ -36,7 +36,7 @@ General options
 Footer
 ------
 
-- **Footer**: Select this checkbox to determine whether the |project_name| footer is shown. If not set, no footer will be visible.
```

Comment:
This variable was here for some reasons (but is probably not used anymore ? - at least not by the project using it some years ago) so no objection to remove.

Reply:
The **|project_name|** variable you mean? It is still defined in the _conf.py_ actually, but it doesn't play well in cases where the variable is used inside some kind of markup (bold, italics, hyperlink etc.). For the sake of consistency, I decided to replace it everywhere. It's not that the project name is likely to change soon and even if it is, it'll be just a matter of doing a (careful) find/replace on "GeoNetwork".